packages/dmn-js-boxed-expression/src/features/literal-expression/components/LiteralExpressionEditorComponent.js

```diff
@@ -0,0 +1,101 @@
+import { Component } from 'inferno';
+
+import { is } from 'dmn-js-shared/lib/util/ModelUtil';
+
+import EditableComponent from 'dmn-js-shared/lib/components/EditableComponent';
+import LiteralExpression from 'dmn-js-shared/lib/components/LiteralExpression';
+
+import { withChangeSupport } from '../../../util/withChangeSupport';
+
+class _LiteralExpressionEditorComponent extends Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this._literalExpression = context.injector.get('literalExpression');
+    this._translate = context.injector.get('translate');
+    this._expressionLanguages = context.injector.get('expressionLanguages');
+    this._variableResolver = context.injector.get('variableResolver', false);
+  }
+
+  getLiteralExpression() {
+    return this.props.expression;
+  }
+
+  editLiteralExpressionText = text => {
+    const literalExpression = this.getLiteralExpression();
+
+    this._literalExpression.setText(literalExpression, text);
+  };
+
+  getEditor() {
+    return this.isFeel() ? FeelEditor : TextEditor;
+  }

```

Comment:
When I use type "java", I still get a FEEL editor. I would expect to get a fallback editor like in the literal expression editor on Decisions

![image](https://github.com/bpmn-io/dmn-js/assets/21984219/ff1416e6-33b7-493c-acb9-77ff42f90ec4)

Reply:
Fixed via https://github.com/bpmn-io/dmn-js/pull/852/commits/28a4210a5a2cf74157fb30f26075e10e064fc4fc